src/cpp/desktop/DesktopGwtCallback.cpp

```diff
@@ -57,8 +57,13 @@ namespace desktop {
 
 namespace {
 WindowTracker s_windowTracker;
+
+#ifdef Q_OS_LINUX
 QString s_globalMouseSelection;
+bool s_clipboardMonitoringEnabled;
```

Comment:
Do we need a static here? (seems like it's recomputed on every access -- is this just a placeholder for a future preference-driven value?)

Reply:
Right; it's effectively just a placeholder. (Currently, we activate this only on RStudio launch; in theory we might want to activate / deactivate it on-the-fly in an active session)

Reply:
Sounds good!